changelog.txt

```diff
@@ -1,3 +1,5 @@
+v.4.0:bla bla
```

Comment:
bla bla? I think this could be improved somewhat

Reply:
More details now...